odoo_tools/tasks/migrate.py

```diff
@@ -0,0 +1,298 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import collections
+import csv
+import json
+import os
+import sys
+
+import git
+import oca_port
+from invoke import task
+
+from ..utils.path import cd
+from .database import get_db_request_result
+
+ANALYZE_SUBMODULES_TO_SKIP = []
+ANALYZE_SUBMODULES = {
```

Comment:
This should be made configurable. Easy option: accept an INI file as cmd line arg.
In the utils you'll find some helper to read ini files.
Or go for a yaml file if you prefer.